jax/_src/prng.py

```diff
@@ -445,6 +445,18 @@ def handler(bufs):
       return PRNGKeyArrayImpl(aval.dtype.impl, phys_handler(bufs))
     return handler
 
+  @staticmethod
+  def make_sharded_array(aval, sharding, arrays, committed, is_out_sharding_from_xla=True):
```

Comment:
I don't think you should take `is_out_sharding_from_xla` as an input here.

If you call it from the public APIs of JAX, this is always False. But it shouldn't even appear here since that is an internal implementation detail.

Reply:
OK - to be clear these are internal APIs, not public APIs. Perhaps I should just hard-code it to True?

Reply:
No, it should be hardcoded to False. Since the sharding is not from XLA. It's coming from the user.

Reply:
Thanks, done